### DISCUSSION TITLE
Custom Forms (simple forms)

### DISCUSSION THREAD

**Discussion by tbugler70** [MasaCMS/MasaCMS#127] [General]
I recently found MASA (after spending years with Mura) and in 7.3 simple forms were deprecated. I've always used simple forms as it gives me a lot more control over from customization and functionality and quite frankly I find creating forms in the form creation tool labour intensive and tedious. Are there are any recommended work arounds for creating custom forms in MASA (my current site has over 50 forms and all are custom -  I know I can still edit them if I import them, but new forms are an issue).

Any thoughts?

Thanks

**Comment by guustnieuwenhuis** [comment]
Hi @tbugler70,

I understand the benefits for you to use simple forms. Would it be an option for you to include the forms via a theme, module or plugin?

**Reply by tbugler70** [reply]
That would be a possible workaround - was their a specific reason i.e. security for deprecating simple forms or perhaps I'm the one of the few that was using it? Perhaps I'll make a project of creating a simple forms plugin.

T.
